docs/development/development-environment-docker/README.md

```diff
@@ -62,15 +92,21 @@ If you want to reset the data you can delete the docker volumes via `docker volu
 
 ## Running tests 
 
-Not all tests are functional within the docker containers yet, so it is recommended to run tests outside of Docker.
-However, you can run tests by executing
+Start all linked containers and migrate the test database first:
 
 ```
-export OPENPROJECT_HOME=`pwd`
+./bin/compose up backend-test
+```
+
+Afterwards, you can start the tests in the running `backend-test` container:
```

Comment:
@b12f could you update the readme for how to run tests? I'd like to try to run them through github action. Thanks!